src/services/todoListing.tsx

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export const accessToken = window.localStorage.getItem("accessToken");
+
+export const getTodoList = () => {
+  return axios.get(`${process.env.REACT_APP_LOGIN_API}/todos`, {
+    headers: { Authorization: `Bearer ${accessToken}` },
```

Comment:
Set the Authorization header through axios interceptors, as we will need it for all the api calls.
https://www.npmjs.com/package/axios#global-axios-defaults

Reply:
interceptors added